tap_stripe/__init__.py

```diff
@@ -125,6 +125,7 @@
 LOGGER = singer.get_logger()
 
 DEFAULT_DATE_WINDOW_SIZE = 30 #days
```

Comment:
```suggestion
DEFAULT_DATE_WINDOW_SIZE = 30 # default date window to fetch newly created records
```

Reply:
Added comment to explain `DEFAULT_DATE_WINDOW_SIZE`